var/spack/repos/builtin/packages/graphite2/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Graphite2(CMakePackage):
+    """Graphite is a system that can be used to create “smart fonts” capable of
```

Comment:
These smart quotes are causing the unit tests to fail

Reply:
Sorry about that, copy/paste bites again. I just pushed a commit to fix this,